lib/utils/utils.go

```diff
@@ -191,14 +191,15 @@ var (
 	ErrFindProjectRoot = errors.New("cannot find project root")
 )
 
-// GetProjectRootPath finds the root of the project where `go.mod` is
+// GetProjectRootPath finds the root of the project where directory `cmd`
+// and subdirectories `gossamer, testcases` are
```

Comment:
to match suggestion below

```suggestion
// and subdirectory `gossamer` are
```